### PR TITLE
Styled index page and renamed submit button

### PR DESCRIPTION
**Pull request by kathtran**

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -16,10 +16,10 @@
         <label for="password">Password:</label>
         <input id="password" type="password" name="password" required maxlength="200" minlength="6" />
       </div>
-      <input type="submit" value="Submit" />
+      <input type="submit" value="Login" />
       <%if (error !== undefined && error){%>
       <!--TODO: use proper CSS class when CSS is complete-->
```
Comment (ibilal) [inline views/index.ejs]: Is this still a TODO? Or is it talking about the two lines below?

Comment (chances) [inline views/index.ejs]: Oh yeah. I think we resolved that.

File: views/index.ejs [hunk 2]
```diff
@@ -16,10 +16,9 @@
         <label for="password">Password:</label>
         <input id="password" type="password" name="password" required maxlength="200" minlength="6" />
```
Comment (ibilal) [inline views/index.ejs]: Can the labels be next to the boxes for input instead of above them?

**Comment by ibilal** [comment]
Other than label placements, LGTM

File: views/index.ejs (hunk 2 above)
Comment (chances) [inline views/index.ejs]: Our above decision was fairly arbitrary. What does everyone else think?

File: public/stylesheets/style.css [hunk 3]
```diff
@@ -1,8 +1,62 @@
 body {
+  display: flex;
```
Comment (RickyV33) [inline public/stylesheets/style.css]: Are we just keeping the background white?

File: views/index.ejs (hunk 2 above)
Comment (RickyV33) [inline views/index.ejs]: I think the labels should be above in input fields. 
Reference: https://uxdesign.cc/design-better-forms-96fadca0f49c#.pp5rd3msc

**Comment by RickyV33** [comment]
Besides for my rebuttal for Iman, LGTM

**Pull request merged**